docs/quickstart.md

```diff
@@ -10,6 +15,15 @@ A local install of [Kustomize](https://github.com/kubernetes-sigs/kustomize/rele
 
 If you are planing to create the simple experiment from this guide, a [minikube](https://kubernetes.io/docs/setup/learning-environment/minikube/) cluster is preferred.
 
+You must have a Kubernetes cluster. Additionally, you will need a local configured copy of kubectl. The Red Sky Ops Tool will use the same configuration as kubectl (usually $HOME/.kube/config) to connect to your cluster.
+
+	1. Client cluster configuration 
+	2. A local installation of Kustomize 
+	3. A client ID and client secret 
+	4. Installation of redskyops to your instance per steps here: https://github.com/redskyops/k8s-experiment/blob/master/docs/install.md
+	5. Server configuration using your client id and client secret here: https://github.com/redskyops/k8s-experiment/blob/master/docs/remote.md
+
```

Comment:
This list is badly formatted, in Markdown, starting a line with three+ spaces indicate a code block so this whole list is rendered as code.

Links should not be rendered as plain text, make linkable text e.g. `\[foo\](some link)`; if you are linking to documents in this repository, use relative links (otherwise they will get messed up when they are published).

All that said; what is the point of this list: we aren't assuming that people reading this have access to the "Enterprise solution" so it isn't likely that they will have the additional information.